egs2/java/asr1/local/score.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Do we need this?
I think we don't need this because this is a monolingual recipe.